quickstarts/microsoft.hybridcompute/machine-windows-configuration/main.bicep

```diff
@@ -0,0 +1,38 @@
+@description('The Azure region where resources in the template should be deployed.')
+param location string
+
+@description('Name of the machine.')
+param machineName string
+
+@description('The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated.')
+param artifactsLocation string = deployment().properties.templateLink.uri
+
+@description('The sasToken required to access _artifactsLocation.')
+@secure()
+param artifactsLocationSasToken string = ''
+
+@description('Name of subfolder containing the configuration package.')
+param configurationFolder string = 'configuration'
+
+@description('File name of the configuration package.')
+param configurationFileName string = 'netcore6.zip'
+
+resource machine 'Microsoft.HybridCompute/Machines@2021-05-20' = {
+  name: machineName
+  location: location
+}
+
+resource configuration 'Microsoft.GuestConfiguration/guestConfigurationAssignments@2020-06-25' = {
+  name: 'netcore6'
+  scope: machine
+  location: location
+  properties: {
+    guestConfiguration: {
+      assignmentType: 'ApplyAndMonitor'
```

Comment:
Any reason why you aren't parameterizing this to allow them to opt into auto correct, too?

Reply:
added